app/src/main/res/values/strings.xml

```diff
@@ -106,9 +106,16 @@
     <!-- Settings strings -->
     <string name="settings_cookies_dialog_title">Clear all cookies and site data</string>
     <string name="settings_cookies_dialog_content2">This will clear all cookies and site data. Clearing this information may sign you out of websites and delete web content that is stored offline.</string>
+    <!-- Text on a button that, when pressed, will clear all user data.
```

Comment:
These strings are actually already translated - I guess we could add these comments though!